canonical-web-and-design/ubuntu.com
# Redesign wizard

## Done

- Implemented [this design](https://app.zeplin.io/project/601d6513b99f24a8412ecfa5/screen/604f8218789166993567404f)
- And [this behaviour](https://www.figma.com/proto/eUNwJ9cJXWaNWjIZlxMLnC/UA-Shop?node-id=23%3A20813&viewport=2224%2C-984%2C0.42681482434272766&scaling=min-zoom)
- Replaced the previous state management by redux in prevision of the react rewrite.

## QA

- Go to https://ubuntu-com-9458.demos.haus/advantage/subscribe?test_backend=true
- Play around with the product selector and verify that it behaves correctly
- Verify that the UI is matching the designs
- Make a purchase using 5555 5555 5555 4444444444.... and check that it goes through correctly

## Issue / Card

Fixes https://github.com/canonical-web-and-design/ubuntu.com/issues/9432 and https://github.com/canonical-web-and-design/web-squad/issues/3900



Blocked because copy is missing from feature coverage section @lechsandecki :
![image](https://user-images.githubusercontent.com/11927929/112475164-ccfe3f00-8d70-11eb-92a4-23f2f7a3cae4.png)
